src/util/expr.cpp

```diff
@@ -19,13 +20,20 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <stack>
 
+/// Move the given argument to the end of `exprt`'s operands.
+/// The argument is destroyed and mutated to a reference to a nil `irept`.
+/// \param expr: `exprt` to append to the operands
 void exprt::move_to_operands(exprt &expr)
 {
   operandst &op=operands();
   op.push_back(static_cast<const exprt &>(get_nil_irep()));
   op.back().swap(expr);
 }
 
+/// Move the given arguments to the end of `exprt`'s operands.
+/// The argument is destroyed and mutated to a reference to a nil `irept`.
```

Comment:
:bulb: "argument is" -> "arguments are"
Could mention that this has the same effect (besides some optimisations) of calling the one-argument version twice.

Reply:
fixed